lib/HttpClient.php

```diff
@@ -68,9 +68,9 @@ public function sendRequest(string $path, ?string $payload, array $extraHeaders
     {
         $protocol = $this->useSsl ? "https://" : "http://";
 
-        $backoff = 100;     // Set initial waiting time to 100ms
+        $backoff = 100; // Set initial waiting time to 100ms
 
-        while ($backoff < $this->maximumBackoffDuration) {
```

Comment:
we may still want the while ,  so that putting 0 actually deactivate backoff ?

Reply:
Replaced by a do/while, so setting `0` at least attempts to send once.

Reply:
ah sorry, I didn't check the whole context, I thought it was a retry middleware :)